crates/typst-library/src/introspection/counter.rs

```diff
@@ -505,38 +485,34 @@ impl Counter {
         Ok(state)
     }
 
-    /// Increases the value of the counter by one.
+    /// カウンター値を1増加。
     ///
-    /// The update will be in effect at the position where the returned content
-    /// is inserted into the document. If you don't put the output into the
-    /// document, nothing happens! This would be the case, for example, if you
-    /// write `{let _ = counter(page).step()}`. Counter updates are always
-    /// applied in layout order and in that case, Typst wouldn't know when to
-    /// step the counter.
+    /// updateは、返り値であるコンテンツが文書中に挿入された位置で適用されます。
+    /// 文書中に出力がなければ何も起こりません！
+    /// 例えば`{let _ = counter(page).step()}`と書いた場合が、この何も起きないときに該当します。
+    /// カウンターの更新は常にレイアウト順に適用され、Typstはいつカウンターをインクリメントするのか知りません。
     #[func]
     pub fn step(
         self,
         span: Span,
-        /// The depth at which to step the counter. Defaults to `{1}`.
+        /// カウンターをインクリメントする深さ。
+        /// デフォルトは`{1}`です。
         #[named]
         #[default(NonZeroUsize::ONE)]
         level: NonZeroUsize,
     ) -> Content {
         self.update(span, CounterUpdate::Step(level))
     }
 
-    /// Updates the value of the counter.
+    /// カウンター値の更新。
     ///
-    /// Just like with `step`, the update only occurs if you put the resulting
-    /// content into the document.
+    /// `step`と同様に、生成されたコンテンツが文書中に配置されたときに限り更新が行われます。
```

Comment:
ひとつ上の`step`のところでは、happens=「起こる」としているので、「起こる」「発生する」「生じる」というようなニュアンスがでると良いと思います。
（最初は「更新されます」とまとめようとしてしまいました。）

```suggestion
    /// `step`と同様に、生成されたコンテンツが文書中に配置されたときに限り更新が起こります。
```